### ISSUE TITLE
other properties???

### ISSUE THREAD

**Issue by 5alidshammout** [wayou/vscode-todo-highlight#190]
https://github.com/wayou/vscode-todo-highlight/blame/4d662c534eb98251c1bba7328a36cba52068d492/README.md#L77
is there more styling properties, were can I find a list for them

**Comment by 5alidshammout** [comment]
can any one help me please

**Comment by samhowle** [comment]
I'd like to know this as well. The main property i'd like to have is to set padding inside the border so that the text doesn't touch up against the border, making it a little harder to read.

**Comment by amoghadami** [comment]
I just add an empty space before and after the text for padding!
like this:

```
"todohighlight.keywords": [
{
  "text": " DONE! ",
  "color": "#fff",
  "backgroundColor": "#22A958"
}
],
  "todohighlight.defaultStyle": {
  "borderRadius": "4px"
}
```
when using, it should have that spaces too. `" DONE! "`

![image](https://user-images.githubusercontent.com/38566991/130213193-e5ace1b4-4b8f-46f3-a9d3-957b09893b62.png)

**Comment by vincent-czi** [comment]
Took me awhile to find it, but there was a link posted in this issue: https://github.com/wayou/vscode-todo-highlight/issues/41
VS Code supports a list of CSS decorations, but it is not all CSS. Here's the available properties:
https://code.visualstudio.com/api/references/vscode-api#ThemableDecorationRenderOptions

@samhowle I was looking for the same kind of thing. Sadly, `padding` is not in the cards, but I was able to achieve _good enough_ with `outline`. See the picture below.

![image](https://user-images.githubusercontent.com/89553795/133836178-4e94fd8a-c037-4578-bb57-1ebf2bdf72a7.png)



Two other things:
- When editing the `outline` value, if you save your settings.json, it DOES NOT re-paint the screen correctly. I don't know why, but you have to close (or at least stop viewing) your settings.json tab, then re-open (or click back into it). Go figure.
- One of the options is `borderSpacing`, but it looks like that just plain does not work. The underlying CSS meaning of it really means something else: it's [about spacing borders from other borders](https://www.w3schools.com/cssref/tryit.asp?filename=trycss_border-spacing). I'm unclear on when/how VS Code would ever make use of it, so not sure why it's exposed  `¯\_(ツ)_/¯`

**Comment by 5alidshammout** [comment]
thanks @vincent-czi

**Comment by samhowle** [comment]
Much appreciated, thanks @vincent-czi!

**Comment by vincent-czi** [comment]
Also, @5alidshammout and @samhowle, you might already know this, but I would recommend taking a look at the "v2" fork over here: https://github.com/jgclark/vscode-todo-highlight (and [here is its VSCode Marketplace page](https://marketplace.visualstudio.com/items?itemName=jgclark.vscode-todo-highlight)). It looks like development here has stopped, but the above links are someone who forked it and is still rolling out new features. I found that briefly after making my post on this issue, and I switched over and am happy with it.
